src/chess/uciloop.cc

```diff
@@ -127,12 +131,37 @@ bool ContainsKey(const std::unordered_map<std::string, std::string>& params,
                  const std::string& key) {
   return params.find(key) != params.end();
 }
+
+#ifdef __EMSCRIPTEN__
+
+extern "C" {
+
+EM_ASYNC_JS(char*, lc0web_get_line, (), {
+  return stringToNewUTF8(String(await globalThis.lc0web_get_line()))
+});
+
+}
+
+bool GetLine(std::string& line) {
+  char *cline = lc0web_get_line();
+  line = cline;
+  free(cline);
+  return true;
+}
+
+#else
+
+bool GetLine(std::string& line) {
+  return std::getline(std::cin, line);
```

Comment:
This breaks the build, I think one solution is to change it to `return !!std::getline(std::cin, line);`.